features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/conf/locales/jaggery/locale_default.json

```diff
@@ -526,6 +531,7 @@
     "Upload API definition as": "Upload API definition as",
     "Upload Certificate": "Upload Certificate",
     "Upload Endpoint Certificates": "Upload Endpoint Certificates",
+    "Upload client certificates" : "Upload client certificates",
```

Comment:
Can we add multiple certs at the same time? Can you attach a UI screen shot?

Reply:
No. We cannot add it . Now I have modified this. Attached updated UI
![screenshot from 2018-10-12 17-11-08](https://user-images.githubusercontent.com/4003116/46867289-1927c400-ce42-11e8-99d9-b193f685f6ba.png)